docs/topics/testing.md

```diff
@@ -7,53 +7,51 @@ You can run the tests in the docker container locally. Testing locally requires
 
 ## To run all
 ```
-make test
+./test.sh
 ```
 
 ## To only run a single suite
 ```
-make test SUITE=repo
+./test.sh repo
 ```
 
 In order to test your own instances of the services, make sure to set the following environment variables to the services that should be tested. 
 
 
 ## To only run a specific file in the repo tests
 ```
-make test SUITE=repo FILTER=babel*
+./test.sh repo --spec babel*
```

Comment:
Yep. I will do that. 

\* Again though this search string and wildcards will once work again as described here, in addition to file path, once we update our WDIO version. The bug is currently here and on `main`. Looking at the `CHANGELOG` on WDIO there there have actually been a pretty good amount of bug fixes since we picked version `8.18.2` in October. We may want to just update it...